README.md

```diff
@@ -78,11 +102,9 @@ After you're finished please remove all the comments and instructions!
 
 <!-- LIVE DEMO -->
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [Live Demo Link](https://francis3400.github.io/Maths-Magician/)
```

Comment:
Great job implemmenting the required change. As suggested by the previous reviewer, kindly deploy your project with either a Github page or Netlify. 😊 

You can follow this [step](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) on how to deploy appropriately because your project is not displaying right now. Check the image below 👇 

<img width="1470" alt="image" src="https://user-images.githubusercontent.com/73854999/221145776-f079ce75-aca4-4a71-87a7-604365085d04.png">